ocaml/xapi/storage_impl.ml

```diff
@@ -371,7 +373,7 @@ module Wrapper = functor(Impl: Server_impl) -> struct
       in
       List.fold_left perform_one vdi_t ops
 
-    let perform_nolock context ~dbg ~dp ~sr ~vdi this_op =
```

Comment:
I think that `vm` should be a required label here, if the new qemu requires us to know the domid.
If we sometimes silently pass it an empty string it may not work correctly.
It would be better to figure out what the correct domid is in each case, even if we don't strictly need it right now, that way we would know that we have a `vm` parameter it is always a valid one.